crates/experimentation_platform/src/api/experiments/types.rs

```diff
@@ -105,7 +105,7 @@ pub enum ContextAction {
 pub struct ContextPutResp {
     pub context_id: String,
     pub override_id: String,
-    pub priority: i32,
+    pub weight: String,
```

Comment:
Weight is string?

Reply:
yes @Datron when it is send as a response or on serialize it will get converted into string as client/frontend wont be able to handle it as a number